front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -133,351 +152,428 @@
+          }
+
+          &:hover:not(.modified) {
+            .rolling-stock {
+              opacity: 0.3;
+            }
+            .action-buttons {
+              display: flex;
             }
           }
         }
+
         &:not(.with-details) {
-          .scenario-timetable-train-with-right-bar {
-            .scenario-timetable-train {
-              .scenario-timetable-train-body,
-              .scenario-timetable-train-tags {
-                display: none;
+          .scenario-timetable-train {
+            .more-info,
+            .scenario-timetable-train-tags {
+              display: none;
+            }
+          }
+        }
+
+        .filter-panel {
+          background-color: var(--ambiantB5);
+          padding: 0 1.438rem 0.25rem 0.313rem;
+          border-top: solid 0.063rem var(--white);
+          border-bottom: solid 0.063rem rgba(0, 0, 0, 0.25);
+          display: flex;
+          flex-direction: column;
+          align-items: end;
+
+          .grid-template {
+            display: grid;
+            grid-template-columns: 1fr 1fr;
+            gap: 0.625rem;
+          }
+
+          .close-filter {
+            transform: translateY(1rem);
+            z-index: 1;
+          }
+
+          .validity-filter {
+            display: flex;
+            margin-bottom: 1rem;
+            font-size: 0.8rem;
+
+            .options-btn {
+              padding: 0.375rem 0.625rem;
+            }
+          }
+
+          .compositions-code {
+            padding: 0.625rem 0.813rem 1rem 1rem;
+            align-self: start;
+            label {
+              height: 1.5rem;
+              line-height: 1.5rem;
+              letter-spacing: 0;
+              text-align: left;
+              vertical-align: top;
+              font-weight: 600;
+            }
+            .composition-tag-filter {
+              display: flex;
+              flex-wrap: wrap;
+              gap: 0.25rem;
+
+              button {
+                border-radius: 0.25rem;
+                border: 0.063rem solid rgba(182, 178, 175, 1);
+                background-color: rgba(250, 249, 245, 1);
+                font-size: 0.875rem;
+                font-weight: 400;
+                text-align: left;
+                line-height: 1.25rem;
+                padding: 0.125rem 0.5rem;
+                margin: 0 0.75rem 0.5rem 0;
+              }
+
+              button:hover {
+                background-color: var(--white);
+                color: var(--black);
+                border: 0.063rem solid var(--black);
+              }
+
+              .selectedTag {
+                background-color: var(--grey60);
+                color: var(--white);
+                border: 0.063rem solid var(--black);
               }
             }
           }
         }
+        .bottom-timetables-trains {
+          height: 1rem;
+          background-color: var(--ambiantB15);
+        }
+        .empty-list {
+          height: 7.5rem;
```

Comment:
This should be `7rem;` and probably a padding-bottom instead of height when you will remove the bottom div

Reply:
Done